target/linux/ath79/image/nand.mk

```diff
@@ -1,4 +1,4 @@
-define Device/glinet_gl-ar300m-common
+define Device/glinet_gl-ar300m-common-nand
```

Comment:
Looks like you changed the wrong commit in rebase, this could have been applied already when moving/adding gl-ar300m ...

Reply:
Should be associated with the "proper" commit with latest push

Reply:
Looks resolved.